Assets/AppCenter/Plugins/AppCenterSDK/Analytics/Shared/Analytics.cs

```diff
@@ -16,6 +16,12 @@ namespace Microsoft.AppCenter.Unity.Analytics
 
     public class Analytics
     {
+        // An event can be lost due to low bandwidth or disk space constraints.     
+        public const int PERSISTENCE_NORMAL = 0x01;
```

Comment:
May be we can move it to separate class e.g. somehow it is done for Native SDKs?
https://github.com/Microsoft/AppCenter-SDK-Android/blob/74f2eb56790d94501a16ebf5be94cfe7fe1e4395/sdk/appcenter/src/main/java/com/microsoft/appcenter/Flags.java